sim/mcuboot-sys/src/c.rs

```diff
@@ -82,6 +86,7 @@ mod raw {
         pub fn invoke_boot_go(sim_ctx: *mut CSimContext, areadesc: *const CAreaDesc) -> libc::c_int;
 
         pub fn boot_trailer_sz(min_write_sz: u8) -> u32;
+        pub fn boot_status_sz(min_write_sz: u8) -> u32;
```

Comment:
Likewise here, `min_write_sz` should also be a `u32`. Hopefully this is consistent with the C code.